example/app/src/App.tsx

```diff
@@ -113,5 +125,21 @@ function bufferToHex(buffer: Uint8Array): string {
     (new Uint8Array(buffer)).forEach((v) => { s += h[v >> 4] + h[v & 15]; });
     return s;
 }
+function bigintToHex(bigintValue: bigint): string {
+    if (typeof bigintValue !== 'bigint') {
+        throw new Error('Input must be a bigint.');
+    }
 
+    const h = '0123456789abcdef';
+    let s = '';
+    let value = bigintValue;
+
+    while (value > 0n) {
+        const digit = Number(value & 15n);
+        s = h[digit] + s;
+        value >>= 4n;
+    }
+
+    return s;
+}
 export default App
```

Comment:
@yaksheshgupta better to add the secondary pool data here also in a separate section.